### ISSUE TITLE
Implement same-line repetition "Message repeated n times" 

### ISSUE THREAD

**Issue by bong0** [cihub/seelog#105]
It would be very helpful to have something like in some java loggers and syslog suppressing lines which are duplicates of each other.
This would require to store the last-logged message and then comparing.

**Comment by goodsign** [comment]
That would only work for asynchronous loggers, right?

**Comment by bong0** [comment]
Yes, I assume that. For strictly non-buffering loggers this cannot work. That's why I pointed it out since I am not so into how seelog works internally or whether there is buffering in that architecture.

**Comment by goodsign** [comment]
Seelog has 4 types of loggers: `sync`, `async loop`, `async timer`, `async adaptive`. This makes sense only for `async loop` logger, probably. As some specific flag.

However I'll think about the possibility of this.

**Comment by vany-egorov** [comment]
+1
ala [ffmpeg](https://ffmpeg.org/ffmpeg-all.html) "Last message repeated n times"

**Comment by bong0** [comment]
@goodsign okay, thank you!
